factory/config.go

```diff
@@ -482,6 +482,19 @@ func compareUPNodesConfigs(u1, u2 map[string]UPNode) (match bool, add, mod, del
 
 }
 
+func compareNetworkSliceInstance(s1, s2 SnssaiInfoItem) (match bool) {
```

Comment:
Can you add intel copyright in this file ?

Reply:
Done